io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/AbstractCSVConfigurationPage.java

```diff
@@ -223,6 +245,13 @@ public Combo getEscape() {
 	}
 
 	/**
+	 * @return the escape
+	 */
+//	public Combo getDecimal() {
```

Comment:
Actually, In the beginning I thought to do something different but couldn't. I wanted to get some idea from Simon regarding ConversionService used to convert data. 
So I have not removed any commented code in my changes.